mockgcp/mockgkemulticloud/attachedcluster.go

```diff
@@ -66,6 +67,28 @@ func (s *GKEMulticloudV1) CreateAttachedCluster(ctx context.Context, req *pb.Cre
 	obj := proto.Clone(req.AttachedCluster).(*pb.AttachedCluster)
 	obj.Name = fqn
 
+	if obj.GetBinaryAuthorization() == nil {
+		obj.BinaryAuthorization = &pb.BinaryAuthorization{
+			EvaluationMode: pb.BinaryAuthorization_DISABLED,
+		}
+	}
+	if obj.GetMonitoringConfig() == nil {
+		obj.MonitoringConfig = &pb.MonitoringConfig{
+			ManagedPrometheusConfig: &pb.ManagedPrometheusConfig{},
+		}
+	}
+	obj.Fleet.Membership = obj.Fleet.Project + "/locations/global/memberships/" + req.AttachedClusterId
+	obj.CreateTime = timestamppb.New(now)
+	obj.UpdateTime = timestamppb.New(now)
+	obj.Etag = "abcdef0123A="
```

Comment:
nit: you can use `computeEtag()` to make it more "real". This field will be handled later on when record the HTTP log.